CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,5 @@
 ### Improvements
 
+- [provider] Support for using [sbt](https://www.scala-sbt.org/)
```

Comment:
We may want to highlight that a Scala example has been added, this is a big deal for us. Thank you!